api.swagger.yaml

```diff
@@ -3998,6 +3998,149 @@ paths:
           description: 'User is not authorized to delete this group category.'
         '404':
           description: 'The UUID does not correspond to an existing group category.'
+  '/group_document_folders':
+    post:
+      tags:
+        - 'group document folders'
+      description: 'Provision a new group document folder.'
+      responses:
+        '201':
+          description: 'The group document folder was created successfully.'
+          headers:
+            Location:
+              description: 'The URL at which the newly provisioned group document folder may be retrieved.'
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/successful-post-response-schema'
+        '400':
+          description: 'The group document folder was not saved due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to create a group document folder.'
+        '422':
+          description: 'The group document folder was not saved due to Drupal validation failure or JSON schema validation failure.'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/group-document-folder-schema-create'
+        required: true
+  '/group_document_folders/list':
+    get:
+      tags:
+        - 'group document folders'
+      description: 'Search for group document folders. An empty array is returned if no group document folders are found.'
+      parameters:
+        - name: group_uuid
+          description: 'Filter by group UUID.'
+          in: query
+          schema:
+            type: string
+        - name: name
+          description: 'Filter by name. Matches on whole words.'
+          in: query
+          schema:
+            type: string
+        - description: 'Filter by external ID.'
+          in: query
+          name: external_id
+          schema:
+            type: string
+        - name: status
```

Comment:
'status' and 'published' should be removed from all group doc folder endpoints as there is no way to change this in the product.

The value will exist in solr/services but as it is not exposed we can assume that all group doc folders will be published (we should still send published: true to services and filter published: true silently between node and solr but just not expose it).

Reply:
done, took it out of the /list filter also.